app/src/main/java/de/westnordost/streetcomplete/quests/barrier_bicycle_barrier_type/BicycleBarrierTypeItem.kt

```diff
@@ -0,0 +1,23 @@
+package de.westnordost.streetcomplete.quests.barrier_bicycle_barrier_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.barrier_bicycle_barrier_type.BicycleBarrierType.*
+import de.westnordost.streetcomplete.view.image_select.Item
+
+fun BicycleBarrierType.asItem() = Item(this, iconResId, titleResId)
+
+private val BicycleBarrierType.titleResId: Int get() = when (this) {
+    SINGLE ->   R.string.quest_barrier_bicycle_type_single
+    DOUBLE ->   R.string.quest_barrier_bicycle_type_double
+    TRIPLE ->   R.string.quest_barrier_bicycle_type_multiple
+    DIAGONAL -> R.string.quest_barrier_bicycle_type_diagonal
+    TILTED ->   R.string.quest_barrier_bicycle_type_tilted
+}
+
+private val BicycleBarrierType.iconResId: Int get() = when (this) {
+    SINGLE ->   R.drawable.barrier_bicycle_barrier_single
+    DOUBLE ->   R.drawable.barrier_bicycle_barrier_double
+    TRIPLE ->   R.drawable.barrier_bicycle_barrier_triple
+    DIAGONAL -> R.drawable.barrier_bicycle_barrier_diagonal
+    TILTED ->   R.drawable.barrier_bicycle_barrier_tilted
+}
```

Comment:
Than having to update both of these two, plus there's no risk of something having a title for SINGLE but forgetting to add the icon for example (but maybe the compile would catch it anyway).

Reply:
> (but maybe the compile would catch it anyway).

yes